samples/bluetooth/hci_usb_h4/sample.yaml

```diff
@@ -10,4 +10,4 @@ tests:
       - usb
       - bluetooth
     # FIXME: exclude due to build error
-    platform_exclude: 96b_carbon stm32l562e_dk
+    platform_exclude: 96b_carbon stm32l562e_dk sensortile_box
```

Comment:
You are adding onto a 'FIXME'. Why is this needed? Can it be resolved properly in some way?

Reply:
This test is related to Bluetooth over USB, and it is performed by Twister automatically when the board has "usb_device" and "ble" capability in its yaml file. However, this dependency for the test is not true that it indicates every board which has aforementioned capabilities should support Bluetooth over USB. At least, at the moment SensorTile.box does not support this feature. In addition, the comment "FIXME" is wrong as there is no problem to be fixed.

I may skip this test without using exclusion since there is already a misspelling in the board's yaml file regarding "usb_device" parameter, and intentionally or by mistake instead of "_" there is a white space.
Please, let me know which approach you prefer.